docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/normalization/rms-norm-15.rst

```diff
@@ -18,14 +18,14 @@ RMS
 *RMSNorm* operation performs Root Mean Square (RMS) normalization on a given input ``data`` along dimensions specified by ``axes`` input.
 `Reference <https://arxiv.org/abs/1910.07467>`__.
 
-.. math::
+.. code-block:: py
```

Comment:
Why is changed from `math`?

Reply:
Just to make it more readable / better highliting